courses/free-courses-te.md

```diff
@@ -70,6 +70,7 @@
 ### C
 
 * [C Language in Telugu - Complete Tutorial in 12 Hours](https://www.youtube.com/watch?v=HdvRHC5TiwE) - Telugu Computer World
+* [C Language Tutorial](https://www.youtube.com/watch?v=FefABtU6SYk) - Telugu Lucky Techzone
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes